.github/workflows/pythonpackage.yml

```diff
@@ -37,9 +37,9 @@ jobs:
     - name: test (pytest)
       run: pytest
     - name: build
-      run: python3 setup.py build sdist bdist_wheel --universal
+      run: python3 -m build
```

Comment:
From my understanding of the docs of build (and I have to admit I really start to hate what python upstream folks are doing in terms of packaging), this would disable building the sdist (the docs refer to a not further defined `default behavior`) build - so I rather would like to have it as explicit as possible here `-s` + `-w` (if that is possible within one run)

Reply:
> build - so I rather would like to have it as explicit as possible here `-s` + `-w`

Yes, you’re right. Also, I have to note that the build library doesn’t have an equivalent for the --universal option in setup.py. This is because PEP 517, which build is based on, expects the build backend to produce wheels with the correct metadata based on the project files and environment (by using setup.cfg or pyproject.toml).